catalog-info.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: pprof-server
+  description: Web server exposing performance profiles of Go services.
+  annotations:
+    github.com/project-slug: segmentio/pprof-server
+    buildkite.com/project-slug: segment/pprof-server
+    segment.io/security-tier: "0"
```

Comment:
Not sure what to make of this, I didn't find any more info here. https://internal-product-docs.twilio.com/docs/taxonomy-platform/backstage/software-catalog-concepts